### PR TITLE
Picker implementation

### PR DESCRIPTION
**Pull request by SteveRobinson99**

### CODE REVIEW

**Comment by SteveRobinson99** [comment]
This implements a picker in place of the text input for 'number of stops'.
When placed next to destination, this free's up a line of space which allows for a larger map. 
All 4 boxes (origin, destination, more trip options and no. of stops) fit into the top two rows and have full functionality - needs testing on other screen sizes.
Map has been enlarged to take advantage of the increased space available.
Currently no conflicts with main.

**Comment by sirbacharach** [comment]
This works perfectly on my phone, although do you think you could add a default option for 0 stops, I know the point of the app is to have a stop so we can check what is nearby, but I'm just thinking that some people might want that option to begin with and then add stops afterwards.

File: components/plan-trip.jsx [hunk 1]
```diff
@@ -76,20 +78,31 @@ const preferences=useContext(PreferencesContext)
...
+    borderColor: preferences.isThemeDark ? 'grey' : 'black', 
+    borderRadius: 20, 
+    overflow: 'hidden',}} >
+        <NumberPicker style={{ width:100, minHeight:20 }} />
+        </View >
+      </View>
       </View>
       
-      <Controller
+      {/* <Controller
```
Comment (dheat23) [inline components/plan-trip.jsx]: think we can remove the commented out Controller now

**Review by dheat23 — COMMENTED** [review]
looks good, just a comment to remove

**Pull request merged**